tests/workflows/test_image_analysis.py

```diff
@@ -0,0 +1,30 @@
+import dask
+import dask.array as da
+import numpy as np
+from dask_image import ndfilters, ndmeasure, ndmorph
+
+
+def test_BBBC039(small_client):
+    images = da.from_zarr(
+        "s3://coiled-datasets/BBBC039", storage_options={"anon": True}
+    )
```

Comment:
I converted the original `.tif` dataset to Zarr format and uploaded them to a public S3 bucket. This had the side-effect of bypassing https://github.com/dask/dask-image/issues/84.